src/auth/guards/jwt-bearer/jwt-bearer.strategy.ts

```diff
@@ -13,14 +15,25 @@ export class JwtBearerStrategy extends PassportStrategy(
   Strategy,
   'jwt-bearer',
 ) {
-  constructor(configService: ConfigService) {
+  constructor(
+    configService: ConfigService,
+    private couchdbService: CouchdbService,
+  ) {
     super({
       jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
       secretOrKey: configService.get<string>(AuthModule.JWT_PUBLIC_KEY),
     });
   }
 
   async validate(data: any): Promise<UserInfo> {
-    return new UserInfo(data.username, data['_couchdb.roles']);
+    const user = await firstValueFrom(
+      this.couchdbService.get('app', data['username']),
+    ).catch(() => {});
+
+    return new UserInfo(
+      data.username,
+      data['_couchdb.roles'],
+      user && user['projects'] ? user['projects'] : [],
+    );
```

Comment:
small code block, but also for future proofing I would suggest we put the UserInfo creation in a service rather than duplicate it.